config/apigroup_config.go

```diff
@@ -0,0 +1,712 @@
+		"azurerm_stream_analytics_reference_input_blob":                                  "streamanalytics",
+		"azurerm_stream_analytics_job":                                                   "streamanalytics",
+		"azurerm_stream_analytics_output_eventhub":                                       "streamanalytics",
+		"azurerm_stream_analytics_stream_input_eventhub":                                 "streamanalytics",
+		"azurerm_stream_analytics_output_servicebus_topic":                               "streamanalytics",
+		"azurerm_subscription":                                                           "subscription",
+		"azurerm_synapse_sql_pool":                                                       "synapse",
+		"azurerm_synapse_role_assignment":                                                "synapse",
+		"azurerm_synapse_sql_pool_extended_auditing_policy":                              "synapse",
+		"azurerm_synapse_linked_service":                                                 "synapse",
+		"azurerm_synapse_workspace_keys":                                                 "synapse",
+		"azurerm_synapse_workspace_extended_auditing_policy":                             "synapse",
+		"azurerm_synapse_workspace_security_alert_policy":                                "synapse",
+		"azurerm_synapse_integration_runtime_azure":                                      "synapse",
+		"azurerm_synapse_managed_private_endpoint":                                       "synapse",
+		"azurerm_synapse_sql_pool_security_alert_policy":                                 "synapse",
+		"azurerm_synapse_integration_runtime_self_hosted":                                "synapse",
+		"azurerm_synapse_sql_pool_vulnerability_assessment":                              "synapse",
+		"azurerm_synapse_workspace_vulnerability_assessment":                             "synapse",
+		"azurerm_synapse_firewall_rule":                                                  "synapse",
+		"azurerm_synapse_workspace":                                                      "synapse",
+		"azurerm_synapse_private_link_hub":                                               "synapse",
+		"azurerm_synapse_spark_pool":                                                     "synapse",
+		"azurerm_iot_time_series_insights_event_source_iothub":                           "timeseriesinsights",
+		"azurerm_iot_time_series_insights_access_policy":                                 "timeseriesinsights",
+		"azurerm_iot_time_series_insights_standard_environment":                          "timeseriesinsights",
+		"azurerm_iot_time_series_insights_gen2_environment":                              "timeseriesinsights",
+		"azurerm_data_lake_store_file":                                                   "txt",
+		"azurerm_app_service_environment_v3":                                             "web",
+		"azurerm_app_service_active_slot":                                                "web",
```

Comment:
Should this group be `appservice` instead of `web`?

Reply:
[`azurerm_app_service_environment_v3`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/app_service_environment_v3) persistent IDs have `Microsoft.Web` as their provider name. This is the reason these two have been grouped under `web`.